README.md

```diff
@@ -51,7 +51,7 @@ Next, lets configure the development server. To keep things simple we can put th
 var hypernova = require('hypernova/server');
 
 hypernova({
-  devMode: true,
+  clustering: false,
```

Comment:
For this one, we're using external clustering solution (`pm2`) in production,
so having `devMode: true` flag would be misleading and confusing for other developers.